transit/script/bart.py

```diff
@@ -0,0 +1,396 @@
+
+
+def station_name():
+    station_name = {
+        "Station": [
+            "12TH",
+            "16TH",
+            "19TH",
+            "24TH",
+            "ANTC",
+            "ASHB",
+            "BALB",
+            "BAYF",
+            "CAST",
+            "CIVC",
+            "COLM",
+            "COLS",
+            "CONC",
+            "DALY",
+            "DBRK",
+            "DELN",
+            "DUBL",
+            "EMBR",
+            "FRMT",
+            "FTVL",
+            "GLEN",
+            "HAYW",
+            "LAFY",
+            "LAKE",
+            "MCAR",
+            "MLBR",
+            "MONT",
+            "NBRK",
+            "NCON",
+            "OAKL",
+            "ORIN",
+            "PCTR",
+            "PHIL",
+            "PITT",
+            "PLZA",
+            "POWL",
+            "RICH",
+            "ROCK",
+            "SANL",
+            "SBRN",
+            "SFIA",
+            "SHAY",
+            "SSAN",
+            "UCTY",
+            "WARM",
+            "WCRK",
+            "WDUB",
+            "WOAK",
+        ],
+        "Node": [
+            16509,
+            16515,
+            16508,
+            16516,
+            15231,
+            16525,
+            16518,
+            16530,
+            16537,
+            16514,
+            16539,
+            16532,
+            16501,
+            16519,
+            16523,
+            16521,
+            16538,
+            16511,
+            16526,
+            16533,
+            16517,
+            16529,
+            16504,
+            16534,
+            16507,
+            16543,
+            16512,
+            16524,
+            16535,
+            16000,
+            16505,
+            15230,
+            16502,
+            16536,
+            16522,
+            16513,
+            16520,
+            16506,
+            16531,
+            16541,
+            16542,
+            16528,
+            16540,
+            16527,
+            16544,
+            16503,
+            16545,
+            16510,
+        ],
+    }
+    df_station_name = pd.DataFrame(station_name)
+    return df_station_name
+
+
+def process_BART_data(file_name, model_run_dir, output_dir, model_BART):
+    # Process BART data for different routes and columns
+    nodes = read_nodes(model_run_dir)
+    station = station_name()
+    BART_BRDA = read_dbf_and_groupby_sum(file_name, "BART", ["A", "TOD"], "AB_BRDA")
+    EBART_BRDA = read_dbf_and_groupby_sum(file_name, "EBART", ["A", "TOD"], "AB_BRDA")
+    OAC_BRDA = read_dbf_and_groupby_sum(file_name, "OAC", ["A", "TOD"], "AB_BRDA")
+
+    BART_XITA = read_dbf_and_groupby_sum(file_name, "BART", ["A", "TOD"], "AB_XITA")
+    EBART_XITA = read_dbf_and_groupby_sum(file_name, "EBART", ["A", "TOD"], "AB_XITA")
+    OAC_XITA = read_dbf_and_groupby_sum(file_name, "OAC", ["A", "TOD"], "AB_XITA")
+
+    # Concatenate and group data
```

Comment:
what's the difference between BART_A and BART_B? (like, why are we building them separately etc?)

Reply:
priority is 👀 if it's a quick fix, can save for later if it's not